nc-news_/src/Components/ArticleList/ArticleCard.jsx

```diff
@@ -0,0 +1,16 @@
+const ArticleCard = ({ article }) => {
+  return (
+    <div>
+      <h2 key={article.id}>{article.title}</h2>
+      <p>
+        Author: {article.author} |Topic: {article.topic} |Posted on:{" "}
+        {new Date(article.created_at).getDate()}/
+        {new Date(article.created_at).getMonth()}/
+        {new Date(article.created_at).getFullYear()}| Comments:{" "}
+        {article.comment_count}
+      </p>
+    </div>
+  );
+};
```

Comment:
I'd consider rendering the article image on your article card as it's very text heavy.